From 489bd334d9ded7e3be770f778d115b8ac9d514d0 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Tue, 17 Apr 2018 01:18:05 +0200
Subject: [PATCH] disable-version-check

Signed-off-by: Markus Volk <f_l_k@t-online.de>

---
 webmin/webmin-lib.pl | 44 ++++++++++++++++++++++----------------------
 1 file changed, 22 insertions(+), 22 deletions(-)

diff --git a/webmin/webmin-lib.pl b/webmin/webmin-lib.pl
index 49f7c86..a14aa08 100755
--- a/webmin/webmin-lib.pl
+++ b/webmin/webmin-lib.pl
@@ -1120,28 +1120,28 @@ my %miniserv;
 &load_theme_library();	# So that UI functions work
 
 # Need OS upgrade
-my %realos = &detect_operating_system(undef, 1);
-if (($realos{'os_version'} ne $gconfig{'os_version'} ||
-     $realos{'os_type'} ne $gconfig{'os_type'}) &&
-    $realos{'os_version'} && $realos{'os_type'} &&
-    &foreign_available("webmin")) {
-	my ($realminor) = split(/\./, $realos{'os_version'});
-	my ($minor) = split(/\./, $gconfig{'os_version'});
-	if ($realos{'os_type'} eq $gconfig{'os_type'} &&
-	    $realminor == $minor) {
-		# Only the minor version number changed - no need to apply
-		&apply_new_os_version(\%realos);
-		}
-	else {
-		# Large enough change to tell the user
-		push(@notifs,
-		    &ui_form_start("$gconfig{'webprefix'}/webmin/fix_os.cgi").
-		    &text('os_incorrect', $realos{'real_os_type'},
-		    		          $realos{'real_os_version'})."<p>\n".
-		    &ui_form_end([ [ undef, $text{'os_fix'} ] ])
-		    );
-		}
-	}
+#my %realos = &detect_operating_system(undef, 1);
+#if (($realos{'os_version'} ne $gconfig{'os_version'} ||
+#     $realos{'os_type'} ne $gconfig{'os_type'}) &&
+#    $realos{'os_version'} && $realos{'os_type'} &&
+#    &foreign_available("webmin")) {
+#	my ($realminor) = split(/\./, $realos{'os_version'});
+#	my ($minor) = split(/\./, $gconfig{'os_version'});
+#	if ($realos{'os_type'} eq $gconfig{'os_type'} &&
+#	    $realminor == $minor) {
+#		# Only the minor version number changed - no need to apply
+#		&apply_new_os_version(\%realos);
+#		}
+#	else {
+#		# Large enough change to tell the user
+#		push(@notifs,
+#		    &ui_form_start("$gconfig{'webprefix'}/webmin/fix_os.cgi").
+#		    &text('os_incorrect', $realos{'real_os_type'},
+#		    		          $realos{'real_os_version'})."<p>\n".
+#		    &ui_form_end([ [ undef, $text{'os_fix'} ] ])
+#		    );
+#		}
+#	}
 
 # Password close to expiry
 my $warn_days = $config{'warn_days'};
