From 98ee74870f5a67d42aee6a67c815632168c3b83b Mon Sep 17 00:00:00 2001
From: Zhai Edwin <edwin.zhai@intel.com>
Date: Sun, 15 Apr 2018 18:49:09 +0200
Subject: [PATCH] Upstream-status: Inappropriate [configuration]

Signed-off-by: Zhai Edwin <edwin.zhai@intel.com>

---
 proftpd/config | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/proftpd/config b/proftpd/config
index a0134c9..2e59da2 100644
--- a/proftpd/config
+++ b/proftpd/config
@@ -1,6 +1,6 @@
-proftpd_path=/usr/local/sbin/proftpd
-proftpd_conf=/usr/local/etc/proftpd.conf
-pid_file=/usr/local/var/proftpd.pid
+proftpd_path=/usr/sbin/proftpd
+proftpd_conf=/etc/proftpd.conf
+pid_file=/var/proftpd.pid
 ftpusers=/etc/ftpusers
 test_config=1
 test_always=0
