From eae1a5c56245817ffba5644f2d93720482dc19c0 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Tue, 2 Sep 2014 00:11:05 -0700
Subject: [PATCH] ajaxterm/ajaxterm/qweb.py: fix hardcode of python2.3

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>

---
 ajaxterm/ajaxterm/qweb.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ajaxterm/ajaxterm/qweb.py b/ajaxterm/ajaxterm/qweb.py
index 20c5092..c658a6b 100644
--- a/ajaxterm/ajaxterm/qweb.py
+++ b/ajaxterm/ajaxterm/qweb.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/env python
 #
 # vim:set et ts=4 fdc=0 fdn=2 fdl=0:
 #
