From caf40a2c62fb890407a1e14e68f4ff39bbbb94a0 Mon Sep 17 00:00:00 2001
From: Zhai Edwin <edwin.zhai@intel.com>
Date: Sun, 15 Apr 2018 18:49:09 +0200
Subject: [PATCH] Upstream-status: Inappropriate [configuration]

Signed-off-by: Zhai Edwin <edwin.zhai@intel.com>

---
 samba/config-generic-linux | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/samba/config-generic-linux b/samba/config-generic-linux
index 8105e0d..5ce1012 100644
--- a/samba/config-generic-linux
+++ b/samba/config-generic-linux
@@ -1,15 +1,15 @@
 list_printers_command=lpc status | grep "[A-z0-9]:" | sed -e 's/://g'
-smb_passwd=/usr/local/samba/private/smbpasswd
+smb_passwd=/usr/bin/smbpasswd
 text_lists=0
 dont_convert=-499
-name_server=/usr/local/samba/bin/nmbd
-smb_conf=/usr/local/samba/lib/smb.conf
-samba_server=/usr/local/samba/bin/smbd
+name_server=/usr/sbin/nmbd
+smb_conf=/etc/samba/smb.conf
+samba_server=/usr/sbin/smbd
 run_from_inetd=0
-samba_password_program=/usr/local/samba/bin/smbpasswd
-samba_status_program=/usr/local/samba/bin/smbstatus
+samba_password_program=/usr/bin/smbpasswd
+samba_status_program=/usr/bin/smbstatus
 swat_path=/usr/local/samba/bin/swat
 sort_mode=0
 smbgroupedit=/usr/local/samba/bin/smbgroupedit
-pdbedit=/usr/local/samba/bin/pdbedit
-net=/usr/local/samba/bin/net
+pdbedit=/usr/bin/pdbedit
+net=/usr/bin/net
