From eaa695cb9123b6a9abce721f4fb80322bba43893 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sat, 11 Nov 2017 14:33:49 +0100
Subject: [PATCH] etckeeper: edit .gitignore

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 update-ignore.d/01update-ignore | 64 +++++++----------------------------------
 1 file changed, 10 insertions(+), 54 deletions(-)

diff --git a/update-ignore.d/01update-ignore b/update-ignore.d/01update-ignore
index 8733a85..9a89299 100755
--- a/update-ignore.d/01update-ignore
+++ b/update-ignore.d/01update-ignore
@@ -105,60 +105,16 @@ writefile () {
 		nl
 	fi
 	
-	comment "old versions of files"
-	ignore "*.old"
-	# Not currently ignored as admins tend to rely on these files.
-	#ignore "passwd-"
-	#ignore "group-"
-	#ignore "shadow-"
-	#ignore "gshadow-"
-	nl
-
-	comment "mount(8) records system state here, no need to store these"
-	ignore blkid.tab
-	ignore blkid.tab.old
-	nl
-	
-	comment "some other files in /etc that typically do not need to be tracked"
-	ignore nologin
-	ignore ld.so.cache
-	ignore prelink.cache
-	ignore mtab
-	ignore mtab.fuselock
-	ignore .pwd.lock
-	ignore "*.LOCK"
-	ignore network/run
-	ignore adjtime
-	ignore lvm/cache
-	ignore lvm/archive
-	ignore "X11/xdm/authdir/authfiles/*"
-	ignore ntp.conf.dhcp
-	ignore .initctl
-	ignore "webmin/fsdump/*.status"
-	ignore "webmin/webmin/oscache"
-	ignore "apparmor.d/cache/*"
-	ignore "service/*/supervise/*"
-	ignore "service/*/log/supervise/*"
-	ignore "sv/*/supervise/*"
-	ignore "sv/*/log/supervise/*"
-	ignore "*.elc"
-	ignore "*.pyc"
-	ignore "*.pyo"
-	ignore "init.d/.depend.*"
-	ignore "openvpn/openvpn-status.log"
-	ignore "cups/subscriptions.conf"
-	ignore "cups/subscriptions.conf.O"
-	ignore "fake-hwclock.data"
-	ignore "check_mk/logwatch.state"
-	nl
-	
-	comment "editor temp files"
-	ignore "*~"
-	ignore ".*.sw?"
-	ignore ".sw?"
-	ignore "#*#"
-	ignore DEADJOE
-
+	/*
+	!.gitignore
+	!neutrino/
+	/neutrino/*
+	!/neutrino/config
+	!network/
+	!auto.*
+	!.etckeeper
+	!minidlna.conf
+	!resolv.conf
 	nl
 	comment "end section $managed_by_etckeeper"
 }
-- 
2.13.3

