From c88a05a8e8ff00263f990f74f5472d65b3cc14d8 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sun, 22 Oct 2017 13:01:41 +0200
Subject: [PATCH] do jpgs for osd+video grabs to improve yweb performance

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 data/y-web/Y_Tools_Screenshot.yhtm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/y-web/Y_Tools_Screenshot.yhtm b/data/y-web/Y_Tools_Screenshot.yhtm
index 5d57d6387..42fa335cc 100644
--- a/data/y-web/Y_Tools_Screenshot.yhtm
+++ b/data/y-web/Y_Tools_Screenshot.yhtm
@@ -51,7 +51,7 @@ function do_snapshot() {
 	if ("{=var-get:grab=}" != "")
 	{
 		if (enableOSD == 1 && enableVideo == 1)
-			dbox_exec_tools("fbshot grab -q -p /tmp/"+filename+".png");
+			dbox_exec_tools("fbshot grab -q -p -j 85 /tmp/"+filename+".png");
 		else if (enableOSD == 1)
 			dbox_exec_tools("fbshot grab -p -o /tmp/"+filename+".png");
 		else if (enableVideo == 1)
-- 
2.13.3

