From 89d814022fc28ce16a66ff23ef9c281e3784c6c6 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Fri, 22 Sep 2017 16:44:01 +0200
Subject: [PATCH] libtuxtext: fix build

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 lib/libtuxtxt/libtuxtxt.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/libtuxtxt/libtuxtxt.cpp b/lib/libtuxtxt/libtuxtxt.cpp
index 40af36282..28faaf8fc 100644
--- a/lib/libtuxtxt/libtuxtxt.cpp
+++ b/lib/libtuxtxt/libtuxtxt.cpp
@@ -13,7 +13,7 @@
 
 #include <sys/ioctl.h>
 #include <fcntl.h>
-
+#include <pthread.h>
 #include "tuxtxt_def.h"
 #include "tuxtxt_common.h"
 int tuxtxt_stop();
-- 
2.13.3

