From 5d5b6679dd75ae751892523b95b0a67add6dd10a Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Fri, 29 Sep 2017 21:59:55 +0200
Subject: [PATCH] fix

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 src/nhttpd/tuxboxapi/controlapi.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/nhttpd/tuxboxapi/controlapi.cpp b/src/nhttpd/tuxboxapi/controla$
index 3a61ec934..bc9d518f2 100644
--- a/src/nhttpd/tuxboxapi/controlapi.cpp
+++ b/src/nhttpd/tuxboxapi/controlapi.cpp
@@ -62,7 +62,7 @@ extern CBouquetManager *g_bouquetManager;
 #elif HAVE_SPARK_HARDWARE
 #define EVENTDEV "/dev/input/nevis_ir"
 #else
-#define EVENTDEV "/dev/input/input0"
+#define EVENTDEV "/dev/input/event0"
 #endif

 //-----------------------------------------------------------------------------
--
2.13.3


