diff --git a/src/gui/infoviewer.cpp b/src/gui/infoviewer.cpp
index b670e58a9..16abc3080 100644
--- a/src/gui/infoviewer.cpp
+++ b/src/gui/infoviewer.cpp
@@ -1169,6 +1169,9 @@ void CInfoViewer::loop(bool show_dot)
 			if (frameBuffer->getActive())
 			{
 				showSNR ();
+				// doesn't belong here, but easiest way to check for a change ...
+				if (is_visible && showButtonBar)
+					infoViewerBB->paint_ca_icons(0);
 				if (timeset) {
 					clock->paint(CC_SAVE_SCREEN_NO);
 				}
diff --git a/src/gui/infoviewer_bb.cpp b/src/gui/infoviewer_bb.cpp
index bd6f7dd2e..3fcb8d453 100644
--- a/src/gui/infoviewer_bb.cpp
+++ b/src/gui/infoviewer_bb.cpp
@@ -807,11 +807,13 @@ void CInfoViewerBB::paint_ca_icons(int notfirst)
 	}
 
 	int caids[] = { 0x5600, 0x900, 0xD00, 0xB00, 0x1800, 0x0500, 0x0100, 0x1700, 0x600, 0x4a00, 0x2600, 0x1000, 0x0E00};
-// 	const char *green = "green"; // unused
+ 	const char *green = "green";
 	const char *white = "white";
 	const char *yellow = "yellow";
 	int icon_space_offset = 0;
 
+	const char *ecm_info_f = "/tmp/ecm.info";
+
 	if(!g_InfoViewer->chanready) {
 		if (g_settings.infobar_casystem_display == 2) {
 			fta = true;
@@ -836,6 +838,40 @@ void CInfoViewerBB::paint_ca_icons(int notfirst)
 	}
 
 	if(!notfirst) {
+		FILE* fd = fopen (ecm_info_f, "r");
+		int ecm_caid = 0;
+		if (fd)
+		{
+			char *buffer = NULL;
+			size_t len = 0;
+			ssize_t read;
+			while ((read = getline(&buffer, &len, fd)) != -1)
+			{
+				if ((sscanf(buffer, "=%*[^9-0]%x", &ecm_caid) == 1) || (sscanf(buffer, "caid: %x", &ecm_caid) == 1))
+				{
+					continue;
+				}
+		}
+			fclose (fd);
+			if (buffer)
+				free (buffer);
+		}
+		if ((ecm_caid & 0xFF00) == 0x1700)
+		{
+			bool nagra_found = false;
+			bool beta_found = false;
+			for(casys_map_iterator_t it = channel->camap.begin(); it != channel->camap.end(); ++it) {
+				int caid = (*it) & 0xFF00;
+				if(caid == 0x1800)
+					nagra_found = true;
+				if (caid == 0x1700)
+					beta_found = true;
+			}
+			if(beta_found)
+				ecm_caid = 0x600;
+			else if(!beta_found && nagra_found)
+				ecm_caid = 0x1800;
+		}
 		for (int i = 0; i < (int)(sizeof(caids)/sizeof(int)); i++) {
 			bool found = false;
 			for(casys_map_iterator_t it = channel->camap.begin(); it != channel->camap.end(); ++it) {
@@ -844,9 +880,9 @@ void CInfoViewerBB::paint_ca_icons(int notfirst)
 					break;
 			}
 			if(g_settings.infobar_casystem_display == 0)
-				paint_ca_icon(caids[i], (found ? yellow : white), icon_space_offset);
+				paint_ca_icon(caids[i], (found ? (caids[i] == (ecm_caid & 0xFF00) ? green : yellow) : white), icon_space_offset);
 			else if(found)
-				paint_ca_icon(caids[i], yellow, icon_space_offset);
+				paint_ca_icon(caids[i], (caids[i] == (ecm_caid & 0xFF00) ? green : yellow), icon_space_offset);
 		}
 	}
 }
