From 59bcd64252bffa0c8be37a73d5c506e22b5741b0 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Mon, 26 Nov 2018 12:23:24 +0100

---
 libavformat/mpegts.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libavformat/mpegts.c b/libavformat/mpegts.c
index 47d8d5f..2ab8402 100644
--- a/libavformat/mpegts.c
+++ b/libavformat/mpegts.c
@@ -1014,10 +1014,12 @@ static int new_pes_packet(PESContext *pes, AVPacket *pkt)
     pes->buffer = NULL;
     reset_pes_packet_state(pes);
 
+    /*
     sd = av_packet_new_side_data(pkt, AV_PKT_DATA_MPEGTS_STREAM_ID, 1);
     if (!sd)
         return AVERROR(ENOMEM);
     *sd = pes->stream_id;
+    */
 
     return 0;
 }
