From c3ef34c0ff68e926834e3d6a6dd20fb52f16b80b Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Wed, 16 Dec 2020 18:47:51 +0100
Subject: [PATCH] sdlmain.cpp: adjust hardcoded path

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 src/gui/sdlmain.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/gui/sdlmain.cpp b/src/gui/sdlmain.cpp
index f4eeda0..b8bc18f 100644
--- a/src/gui/sdlmain.cpp
+++ b/src/gui/sdlmain.cpp
@@ -986,7 +986,7 @@ Bitu GFX_SetSize(Bitu width,Bitu height,Bitu flags,double scalex,double scaley,G
 
 	char tmpBuf[1024];
 	sprintf(tmpBuf, "Using DOS framebuffer size of %d x %d", width, height);
-	OutputDebugString(tmpBuf);
+//	OutputDebugString(tmpBuf);
 
 	sdl.draw.width=width;
 	sdl.draw.height=height;
@@ -1897,7 +1897,7 @@ inline const char *os_separator()
 #if defined (WIN32)
 const char* sg_pathToShaders = ".\\Shaders\\";
 #else
-const char* sg_pathToShaders = "/usr/local/share/dosbox/";
+const char* sg_pathToShaders = "/usr/share/dosbox/";
 #endif // defined
 
 
@@ -3056,7 +3056,7 @@ int main(int argc, char* argv[]) {
 #endif
 #ifdef __WINDOWS__
 	SDL_setenv("SDL_AUDIODRIVER", "directsound", 1);
-#endif
+#endif
 
 	if ( SDL_Init( SDL_INIT_AUDIO|SDL_INIT_VIDEO|SDL_INIT_TIMER|SDL_INIT_EVENTS) < 0 )
 		E_Exit("Can't init SDL %s",SDL_GetError());
-- 
2.28.0

