From e38cd454e11796fe828f0fb68c41abb27462ef4f Mon Sep 17 00:00:00 2001
From: "Roy.Li" <rongqing.li@windriver.com>
Date: Fri, 25 Jun 2021 18:36:17 +0000
Subject: [PATCH] Enable tcp_wrapper.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Roy.Li <rongqing.li@windriver.com>

---
 builddefs.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/builddefs.h b/builddefs.h
index e908352..0106d1a 100644
--- a/builddefs.h
+++ b/builddefs.h
@@ -1,7 +1,7 @@
 #ifndef VSF_BUILDDEFS_H
 #define VSF_BUILDDEFS_H
 
-#undef VSF_BUILD_TCPWRAPPERS
+#define VSF_BUILD_TCPWRAPPERS
 #define VSF_BUILD_PAM
 #undef VSF_BUILD_SSL
 
