From 5e1d264ff459cdb2c33473eafa11a6405126ea24 Mon Sep 17 00:00:00 2001
From: "[CST] Focus" <focus.cst@gmail.com>
Date: Sat, 24 Aug 2013 10:03:45 +0400
Subject: [PATCH] xupnpd: don't bind daemon to specific ip address

---
 src/luaxcore.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/luaxcore.cpp b/luaxcore.cpp
index 8aa38d8..6fa1112 100644
--- a/luaxcore.cpp
+++ b/luaxcore.cpp
@@ -129,8 +129,10 @@ namespace core
         sin.sin_addr.s_addr=INADDR_ANY;
         sin.sin_port=htons(port);
 
+#if 0
         if(strcmp(host,"*") && strcmp(host,"any"))
             sin.sin_addr.s_addr=inet_addr(host);
+#endif
 
         int fd=socket(sin.sin_family,SOCK_STREAM,0);
         if(fd==-1)
